### PR TITLE
Add docs for tabBarOnLongPress and animated switch navigator

### PR DESCRIPTION
**Pull request by jarvisluong**
- Added docs for tabBarOnLongPress (https://github.com/react-navigation/react-navigation.github.io/issues/384)
- Added API section for `createAnimatedSwitchNavigator` and updated the auth-flow guide to enable animations.

### CODE REVIEW

File: docs/animated-switch-navigator.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: animated-switch-navigator
+title: createAnimatedSwitchNavigator
+sidebar_label: createAnimatedSwitchNavigator
+---
+
+An SwitchNavigator with animation support.
```
Comment (brentvatne) [inline docs/animated-switch-navigator.md]:
```suggestion
A switch navigator with the ability to animate transitions between screens. 
```

File: docs/animated-switch-navigator.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## API Definition
+
+```js
+import createAnimatedSwitchNavigator from 'react-navigation-animated-switch';
+
+createAnimatedSwitchNavigator(RouteConfigs, SwitchNavigatorConfig);
+```
+
+The route configs and navigator configs are identical to [createSwitchNavigator](switch-navigator.html)
```
Comment (brentvatne) [inline docs/animated-switch-navigator.md]: not exactly true, you a `transition` option as described in https://github.com/react-navigation/react-navigation-animated-switch#usage

**Comment by brentvatne** [comment]
thanks! it's usually best to split separate changes up into different prs but that's fine for now. left a couple of comments

**Comment by jarvisluong** [comment]
thanks for the feedback! I have updated the docs for `transition` option as well.

**Pull request merged**